app/src/main/java/com/example/learn09082021v001/CustomView_group.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.learn09082021v001
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.Button
+import android.widget.EditText
+
+import android.widget.LinearLayout
+import kotlinx.android.synthetic.main.view_sub_bottom.view.*
+
+
+class CustomView_group (context: Context, attrs: AttributeSet,) :
```

Comment:
why there is a comma after `AttributeSet`?

Reply:
local project done have comma affter